Gemfile.lock

```diff
@@ -1,3 +1,22 @@
+GIT
+  remote: git@github.com:wordpress-mobile/release-toolkit
+  revision: 664f5f3e654abc0eb67b7b4650bb560731041e6f
```

Comment:
This is the latest commit after the merge of https://github.com/wordpress-mobile/release-toolkit/pull/309.

I don't plan to create an official new release of the toolkit until I finish all the work at least on the Widget L10n project (ref: paaHJt-2J8-p2): doesn't seem worth it until we have everything in place and cleaned up.